netmiko/vyos/vyos_ssh.py

```diff
@@ -131,10 +131,12 @@ def send_config_set(
         )
 
     def save_config(
-        self,
-        cmd: str = "copy running-config startup-config",
-        confirm: bool = False,
-        confirm_response: str = "",
+        self, cmd: str = "save", confirm: bool = False, confirm_response: str = ""
     ) -> str:
-        """Not Implemented"""
-        raise NotImplementedError
+        """Saves Config."""
+        output = self._send_command_str(command_string=cmd)
```

Comment:
You should be able to just do the following:

```python
        output = return super().save_config(
            cmd=cmd, confirm=confirm, confirm_response=confirm_response
        )
        if "Done" not in output:
            raise ValueError(f"Save failed with following errors:\n\n{output}")
        return output

The `save` command will be sent to the remote device in the parent method (i.e. as part of that super() call).

Reply:
Your recommendation caused an error:
```
```
    def save_config(
        self, cmd: str = "save", confirm: bool = False, confirm_response: str = ""
    ) -> str:
        """Saves Config."""
        output = self._send_command_str(command_string=cmd)
        return super().save_config(
            cmd=cmd, confirm=confirm, confirm_response=confirm_response
        )
        if "Done" not in output:
            raise ValueError(f"Save failed with following errors:\n\n{output}")
        return output
```

Reply:
Ooops I screwed up the syntax above :-)